src/mlpack/methods/ann/layer/layer.hpp

```diff
@@ -25,6 +25,8 @@
 #include "linear.hpp"
 #include "linear_no_bias.hpp"
 #include "lstm.hpp"
+#include "max_pooling.hpp"
+#include "mean_pooling.hpp"
```

Comment:
Do we need really need to add these headers? Let me know your thoughts on this. Maybe I missed something.

Reply:
I thought it was somehow missed?